server/src/main/java/io/spine/server/entity/storage/Columns.java

```diff
@@ -258,19 +256,19 @@ private static boolean isEntityColumn(Method getter) {
     }
 
     /**
-     * Obtains the method with the same signature as the specified method from the specified class.
+     * Obtains the method from the specified class with the same signature as the specified method.
      *
+     * @param target the class to obtain the method with the signature
      * @param method the method to get the signature
-     * @param aClass the class to obtain the method with the same signature
      * @return the method with the same signature obtained from the specified class
      */
-    private static Optional<Method> getMethod(Method method, Class<?> aClass) {
+    private static Optional<Method> getMethod(Class<?> target, Method method) {
```

Comment:
Can the result method be non-`public`? 
Please check and add a corresponding test.

Reply:
Done.